### PR TITLE
pacman.cpp: Added Miss Packman Plus.

### PR DESCRIPTION
**Pull request by mdeslaur**
This adds support for the Miss Packman Plus roms that have the alternate mazes.

### CODE REVIEW

File: src/mame/pacman/pacman.cpp [hunk 1]
```diff
@@ -8234,6 +8274,7 @@ GAME( 1981, pacgal,     mspacman, woodpek,  mspacman, pacman_state,  empty_init,
 GAME( 1981, mspacpls,   mspacman, woodpek,  mspacman, pacman_state,  empty_init,    ROT90,  "hack",                                  "Ms. Pac-Man Plus",                                 MACHINE_SUPPORTS_SAVE )
 GAME( 1992, mschamp,    mspacman, mschamp,  mschamp,  pacman_state,  init_mschamp,  ROT90,  "hack",                                  "Ms. Pacman Champion Edition / Zola-Puc Gal",       MACHINE_SUPPORTS_SAVE ) // Rayglo version
 GAME( 1995, mschamps,   mspacman, mschamp,  mschamp,  pacman_state,  init_mschamp,  ROT90,  "hack",                                  "Ms. Pacman Champion Edition / Super Zola-Puc Gal", MACHINE_SUPPORTS_SAVE )
+GAME( 1981, mspacpl2,   mspacman, woodpek,  mspacman, pacman_state,  init_mspacpl2, ROT90, "hack",                                  "Miss Packman Plus", MACHINE_SUPPORTS_SAVE )
```
Comment (0kmg) [inline src/mame/pacman/pacman.cpp:8277]: If you could make the last three columns starting at "hack" line up with the rows above it, that would be great. Thanks!

Comment (mdeslaur) [reply src/mame/pacman/pacman.cpp]: whoops, sure!

**Comment by mdeslaur** [comment]
FYI, I sent the roms that go with this commit to the code email address.
The rom checksums also match the first board listed here: http://lawnmowerman.rotheblog.com/#bootlegs

File: src/mame/pacman/pacman.cpp [hunk 2]
```diff
@@ -8234,6 +8274,7 @@ GAME( 1981, pacgal,     mspacman, woodpek,  mspacman, pacman_state,  empty_init,
 GAME( 1981, mspacpls,   mspacman, woodpek,  mspacman, pacman_state,  empty_init,    ROT90,  "hack",                                  "Ms. Pac-Man Plus",                                 MACHINE_SUPPORTS_SAVE )
 GAME( 1992, mschamp,    mspacman, mschamp,  mschamp,  pacman_state,  init_mschamp,  ROT90,  "hack",                                  "Ms. Pacman Champion Edition / Zola-Puc Gal",       MACHINE_SUPPORTS_SAVE ) // Rayglo version
 GAME( 1995, mschamps,   mspacman, mschamp,  mschamp,  pacman_state,  init_mschamp,  ROT90,  "hack",                                  "Ms. Pacman Champion Edition / Super Zola-Puc Gal", MACHINE_SUPPORTS_SAVE )
+GAME( 1981, mspacpl2,   mspacman, woodpek,  mspacman, pacman_state,  init_mspacpl2, ROT90,  "hack",                                  "Miss Packman Plus",                                MACHINE_SUPPORTS_SAVE )
```
Comment (cuavas) [inline src/mame/pacman/pacman.cpp:8277]: Maybe make the short name `mspackpls` or something, reflecting the “Packman” spelling on the title screen?

Comment (mdeslaur) [reply src/mame/pacman/pacman.cpp:8277]: Isn't the short name limited to 8 characters? Also, it is related to mspacpls as that is a hacked romset that lifted the levels from this one...

Comment (cuavas) [reply src/mame/pacman/pacman.cpp:8277]: They’re limited to 16 characters.  We often make clones the name of the parent with a suffix appended.

Comment (mdeslaur) [reply src/mame/pacman/pacman.cpp]: Cool, renamed to mspackpls, thanks.

**Pull request merged**